app/src/main/java/com/internship/move/feature/splash/SplashFragment.kt

```diff
@@ -1,20 +1,36 @@
 package com.internship.move.feature.splash
 
+import android.content.Context
 import android.os.Bundle
 import android.os.Handler
 import android.os.Looper
 import android.view.View
 import androidx.fragment.app.Fragment
 import androidx.navigation.fragment.findNavController
 import com.internship.move.R
+import com.internship.move.feature.authentification.register.RegisterFragment
+import com.internship.move.feature.map.MapFragment
 
 class SplashFragment : Fragment(R.layout.fragment_splash) {
 
+
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
+        val sharedPref = activity?.getPreferences(Context.MODE_PRIVATE)
+
         Handler(Looper.getMainLooper()).postDelayed(
-            { findNavController().navigate(SplashFragmentDirections.actionSplashFragmentToOnboardingGraph()) },
+            {
+                when (sharedPref?.getString(
```

Comment:
`when` is a bit confusing. Don't use resource strings in this case.

See my comment from above and change it.

Reply:
Cata told me to use when in this case 😄